gwcs/tests/test_wcs.py

```diff
@@ -514,3 +514,73 @@ def test_to_fits_sip():
     with pytest.raises(ValueError):
         miriwcs.bounding_box = None
         mirisip = miriwcs.to_fits_sip(bounding_box=None, max_inv_pix_error=0.1)
+
+
+def test_to_fits_tab_cube(gwcs_3d_galactic_spectral):
```

Comment:
I'm missing the magic here. How does this get called?

Reply:
It is a `"fixture"` defined in `conftest.py`. See https://docs.pytest.org/en/latest/fixture.html